JobSchema.md

```diff
@@ -0,0 +1,221 @@
+Job Schema Service
+==================
+
+#### Table of Contents
+_______
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+- [Response structure](#response-structure)
+- [Versioning](#versioning)
+
+## Summary
+
+Job Schema Service (JSS) takes a request containing data for a job posting and returns a response conforming to the [schema.org](﻿http://schema.org/) [JobPosting schema](﻿http://schema.org/JobPosting).
```

Comment:
in preview mode, these links don't seem to be working. also (beef comment), I might just link the whole thing to /JobPosting instead of having the first part link to the schema.org homepage?

Reply:
I'm not sure why it's not working in preview but that is how you do a link in markdown: [a link](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links). I'll double check it after it gets merged.

Reply:
yeah, my suspicion was that it was failing due to no www, but your example link in that comment works without it. maybe it's just a preview thing.